sphinx/tutorial/cylc/runtime/introduction.rst

```diff
@@ -0,0 +1,388 @@
+interface (the cylc GUI) which can be used for monitoring and
+interaction.
+
+The cylc GUI looks quite like ``cylc graph`` but the tasks are colour-coded to
+represent their state as in the following diagram.
+
+.. digraph:: example
+   :align: center
+
+   bgcolor=none
+
+   Waiting [color="#88c6ff"]
+   Running [style="filled" color="#00c410"]
+   Succeeded [style="filled" color="#ada5a5"]
+
+.. minicylc::
+   :align: center
+
+    a => b => c
+    b => d => f
+    e => f
+
+This is the "graph view". The cylc GUI has two other views called "tree" and
+"dot".
+
+.. figure:: ../img/cylc-gui-graph.png
+   :figwidth: 50%
+   :align: center
+
+   Screenshot of the cylc GUI in "Graph View" mode.
+
+.. figure:: ../img/cylc-gui-tree.png
+   :figwidth: 50%
+   :align: center
+
+   Screenshot of the cylc GUI in "Tree View" mode.
+
+.. figure:: ../img/cylc-gui-dot.png
+   :figwidth: 50%
+   :align: center
+
+   Screenshot of the cylc GUI in "Dot View" mode.
+
+
+Where Do All The Files Go?
+--------------------------
+
+The Work Directory
+^^^^^^^^^^^^^^^^^^
+
+When a :term:`task` is run cylc creates a directory for the :term:`job` to run
+in, this is called the :term:`work directory`.
+
+By default the work directory is in a directory structure under the
+:term:`cycle point` and the :term:`task` name:
+
+.. code-block:: sub
+
+   ~/cylc-run/<suite-name>/work/<cycle-point>/<task-name>
+
+The Job Log Directory
+^^^^^^^^^^^^^^^^^^^^^
+
+When a task is run cylc generates a :term:`job script`, this is stored in the
+:term:`job log directory` as the file ``job``.
+
+When the :term:`job script` is executed the stdout and stderr are redirected
+into the ``job.out`` and ``job.err`` files which are also stored in the
+:term:`job log directory`.
+
+The :term:`job log directory` lives in a directory structure under the
+:term:`cycle point`, :term:`task` name and the :term:`job submission number`:
+
+.. code-block:: sub
+
+   ~/cylc-run/<suite-name>/log/job/<cycle-point>/<task-name>/<job-submission-num>/
+
+The :term:`job submission number` starts at 1 and increments each time a task
+is re-run.
+
+.. tip::
+
+   If a task has run and is still visible in the cylc GUI you can view its
+   :term:`job log files <job log>` by right-clicking on the task and selecting
+   "View".
+
+   .. image:: ../img/cylc-gui-view-log.png
+      :align: center
+      :scale: 75%
+
+Suite Files
+^^^^^^^^^^^
+
+Along with the :term:`work directory` and :term:`job log directory`, cylc
+generates other files and directories when it runs a suite:
+
+``cylc-suite.db``
+   A symlink to the database which cylc uses to record the state of the suite.
+``log/``
+   The directory in which various log files, including
+   :term:`job log files <job log>` live.
+``suite.rc.processed``
+   A copy of the ``suite.rc`` file made after any `Jinja2`_ has been processed
+   - we will cover this in the :ref:`tutorial-cylc-consolidating-configuration`
+   section.
```

Comment:
Also worth knowing `log/suiterc/` for a full list.

Reply:
Restarts aren't introduced until the next section.